x-pack/plugins/security_solution/server/lib/entity_analytics/risk_engine/routes/init.ts

```diff
@@ -43,16 +48,20 @@ export const riskEngineInitRoute = (
         const initResult = await riskEngineDataClient.init({
           taskManager,
           namespace: spaceId,
+          isAssetCriticalityEnabled,
+          assetCriticalityDataClient,
+          riskScoreDataClient,
         });
 
-        const initResultResponse = {
+        const initResultResponse: InitRiskEngineResultResponse = {
           risk_engine_enabled: initResult.riskEngineEnabled,
           risk_engine_resources_installed: initResult.riskEngineResourcesInstalled,
           risk_engine_configuration_created: initResult.riskEngineConfigurationCreated,
           legacy_risk_engine_disabled: initResult.legacyRiskEngineDisabled,
           errors: initResult.errors,
         };
 
+        // TODO: after enabling asset criticality feature, we don't need assetCriticalityInstallFailed
```

Comment:
Your refactor hid [this comment](https://github.com/elastic/kibana/pull/171324#discussion_r1399842269), but I think it's still relevant. What's needed to remove this comment?

Reply:
removed!